ui/src/UI/Resources/Localization/Translations/en-us.ftl

```diff
@@ -81,7 +81,7 @@ tray-menu-hide = _Hide
 settings-auto-launch = Launch VPN app on computer startup
 settings-unsecured-network-title = Unsecured network alert
 settings-unsecured-network-content = Get notified if you connect to an unsecured Wi-Fi network
-settings-local-device-access-title = Allow access to your local network
+settings-local-device-access-title = Local network access
```

Comment:
This might fail the UI test. Please update the same string in UI tests.

Reply:
@connected-mhuang Could you point them in the right direction?

Reply:
@tonymurphy-rgb Could you please update this line as well? https://github.com/mozilla-services/guardian-vpn-windows/blob/680ff1cff71b2dc5b8927bd5e5241ba1b32b2551/test/smoke/FirefoxPrivateVPNUITest/FirefoxPrivateVPNUITest/Tests/SettingTest.cs#L89